R/final_size.R

```diff
@@ -1,54 +1,77 @@
 #' Calculate final epidemic size
 #'
-#' This function calculates final epidemic size using SIR model for heterogeneously mixing population
+#' This function calculates final epidemic size using SIR model for
+#' heterogeneously mixing population
 #' @param r0  Basic reproduction number. Default is 2
-#' @param contact_matrix  Social contact matrix. Entry mm_ij gives average number of contacts in group i reported by participants in group j
-#' @param demography_vector  Demography vector. Entry pp_i gives proportion of total population in group i (model will normalise if needed)
-#' @param prop_suscep  Proportion of each group susceptible. Null assumption is fully susceptible
+#' @param contact_matrix  Social contact matrix. Entry $mm_{ij}$ gives average
+#' number of contacts in group $i$ reported by participants in group j
+#' @param demography_vector  Demography vector. Entry $pp_{i}$ gives proportion
+#' of total population in group $i$ (model will normalise if needed)
+#' @param prop_suscep  Proportion of each group susceptible. Null assumption is
+#' fully susceptible
 #' @keywords epidemic model
 #' @export
 #' @examples
-#' final_size()
+#' library(socialmixr)
+#' data(polymod)
+#' contact_data <- contact_matrix(polymod, countries = "United Kingdom", age.limits = c(0,20,40))
+#' 
+#' c_matrix <-t(contact_data$matrix) # Define contact matrix (entry ij is contacts in group i reported by group j)
+#' d_vector <- contact_data$participants$proportion # Define proportion in each age group
+#' p_suscep <- c(1,0.5,0.5) # Define proportion of age group that is susceptible to infection
+#' 
+#' # Run final size model
+#' final_size(r0=2,contact_matrix = c_matrix, demography_vector = d_vector, prop_suscep = p_suscep)
```

Comment:
```suggestion
#' final_size(r0, contact_matrix = c_matrix, demography_vector = d_vector, prop_suscep = p_suscep)
```

Reply:
It may be risky to leave a default for the value of R0. Also, do we want to make an exception re capitalization of arguments to distinguish _R0_ (the basic reproduction number) from the daily growth rate _r_?

Reply:
@thibautjombart Have accepted and committed, but the `r0` vs `R0` naming issue will recur, so feel free to make a policy and I'll go with it!